src/components/select/filterable-select/filterable-select.spec.js

```diff
@@ -369,23 +370,88 @@ describe('FilterableSelect', () => {
       document.body.removeChild(domNode);
     });
   });
-});
 
+  describe('when the onKeyDown prop is passed', () => {
+    const expectedEventObject = {
+      key: 'ArrowDown'
+    };
 
-describe('when the onKeyDown prop is passed', () => {
-  const expectedEventObject = {
-    key: 'ArrowDown'
-  };
+    it('then when a key is pressed, that prop should be called with expected values', () => {
```

Comment:
I think that we should to check also if the list appears after using `arrowDown`

Reply:
that check is already there on line 80:
https://github.com/Sage/carbon/pull/3271/files/316cd4633c565dca6b59b00513c797f78b144173#diff-430f630553c33723423ecadaccd9f19bL81